pkg/config/config.go

```diff
@@ -494,8 +502,12 @@ var DefaultConfig = Config{
 			{Mime: webrtc.MimeTypeVP9},
 			{Mime: webrtc.MimeTypeAV1},
 		},
-		EmptyTimeout:                 5 * 60,
-		DepartureTimeout:             20,
+		EmptyTimeout:     5 * 60,
+		DepartureTimeout: 20,
+	},
+	Limit: LimitConfig{
+		MaxMetadataSize:              128000,
+		MaxAttributesSize:            128000,
```

Comment:
256kb is a lot... does this set unrealistic expectations given the efficiency of implementation?

Reply:
was just thinking we'd set it the same as metadata.. would be happy to adjust back to 64k